src/fold/loop/encase.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING, Any, Callable, Dict, Optional, Tuple, Union
+
+import pandas as pd
+from sklearn.metrics import mean_squared_error
+
+from ..all_types import OutOfSamplePredictions, TransformationsOverTime
+from ..splitters import ExpandingWindowSplitter, Splitter
+from ..transformations.base import BlocksOrWrappable
+from .backtesting import backtest
+from .training import train
+from .types import Backend, TrainMethod
+
+if TYPE_CHECKING:
+    from krisi import ScoreCard
+
+
+def backtest_score(
+    transformations_over_time: TransformationsOverTime,
+    X: pd.DataFrame,
+    y: pd.Series,
+    splitter: Splitter,
+    backend: Backend = Backend.no,
+    sample_weights: Optional[pd.Series] = None,
+    silent: bool = False,
+    mutate: bool = False,
```

Comment:
sorry about it. this is something we really shouldn't expose, it's really for internal use :)

Reply:
Which ones? `mutate`?